src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -63,7 +64,7 @@ protected static class SidStatus {
     private Optional<LoginData> loginData;
 
     private final Gson gson = new Gson();
-
+    private final CaptchaAPI captchaSolver;
```

Comment:
avoid duplicating state, simply use CaptchaAPI.getInstance() wherever you actually need it